types/index.d.ts

```diff
@@ -189,7 +189,7 @@ export class Type {
   readonly name: string | undefined;
   readonly branchName: string | undefined;
   readonly typeName: string;
-  static forSchema(schema: Schema, opts?: Partial<ForSchemaOptions>): Type;
+  static forSchema(schema: Schema | Schema[], opts?: Partial<ForSchemaOptions>): Type;
```

Comment:
Unfortunately this will cause `forSchema` to accept invalid inputs. `Schema` already contains array types (`DefinedType[]` and `Type[]`), and nested arrays should not be allowed.

Reply:
My code had:
```
Avro.Type.forSchema([Avro.Type.forSchema('int'), Avro.Type.forSchema('string')])
```
which works with the type change but fails with the current definitions due to `Type` not assignable to `DefinedType`.

I think this should have been:
```
Avro.Type.forTypes([Avro.Type.forSchema('int'), Avro.Type.forSchema('string')])
```
instead, possibly?